modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/buffer/IndexerRequestBufferOverflowHandler.java

```diff
@@ -58,42 +50,17 @@ public void bufferOverflowed(
 		}
 	}
 
-	@Activate
-	@Modified
-	protected void activate(Map<String, Object> properties) {
-		IndexerRegistryConfiguration indexerRegistryConfiguration =
-			ConfigurableUtil.createConfigurable(
-				IndexerRegistryConfiguration.class, properties);
-
-		float minimumBufferAvailabilityPercentage =
-			indexerRegistryConfiguration.minimumBufferAvailabilityPercentage();
-
-		if ((minimumBufferAvailabilityPercentage > 1) ||
-			(minimumBufferAvailabilityPercentage < 0.1)) {
-
-			if (_log.isWarnEnabled()) {
-				_log.warn(
-					StringBundler.concat(
-						"Invalid minimum buffer availability percentage: ",
-						minimumBufferAvailabilityPercentage,
-						", using default value",
-						_DEFAULT_MINIMUM_BUFFER_AVAILABILITY_PERCENTAGE));
-			}
-		}
+	public void setMinimumBufferAvailabilityPercentage(
+		float minimumBufferAvailabilityPercentage) {
 
 		_minimumBufferAvailabilityPercentage =
 			minimumBufferAvailabilityPercentage;
 	}
 
-	private static final float _DEFAULT_MINIMUM_BUFFER_AVAILABILITY_PERCENTAGE =
-		0.90F;
-
 	private static final Log _log = LogFactoryUtil.getLog(
 		IndexerRequestBufferOverflowHandler.class);
 
-	@Reference
-	private IndexerRequestBufferExecutor _indexerRequestBufferExecutor;
-
+	private final IndexerRequestBufferExecutor _indexerRequestBufferExecutor;
 	private volatile float _minimumBufferAvailabilityPercentage;
```

Comment:
Hey @regiCesar21, what is the initial value of the `_minimumBufferAvailabilityPercentage` after your changes? Thx